sovrin_node/server/node.py

```diff
@@ -365,12 +379,19 @@ def processRequest(self, request: Request, frm: str):
             self.send_ack_to_client(request.key, frm)
             result = self.reqHandler.handleGetClaimDefReq(request, frm)
             self.transmitToClient(Reply(result), frm)
+        elif request.operation[TXN_TYPE] == GET_TXNS:
+            super().processRequest(request, frm)
         else:
             # forced request should be processed before consensus
-            if request.operation[TXN_TYPE] == POOL_UPGRADE and request.isForced():
+            if (request.operation[TXN_TYPE] in [POOL_UPGRADE, POOL_CONFIG]) and request.isForced():
                 self.configReqHandler.validate(request)
                 self.configReqHandler.applyForced(request)
-            super().processRequest(request, frm)
+            #here we should have write transactions that should be processed only on writable pool
+            if self.poolCfg.isWritable() or (request.operation[TXN_TYPE] == POOL_CONFIG):
+                super().processRequest(request, frm)
```

Comment:
Do we have tests to check that read (GET) request can be processed regardless of writeable state?